### ISSUE TITLE
Can't start Firefox through Geckodriver on a version below 0.30.0 on Selenium 4.0.0 RC 2

### ISSUE THREAD

**Issue by AlexWFreitas** [SeleniumHQ/selenium#9880]
## 💥 Regression Report

On Selenium 4 Rc1 I was able to use Geckodriver 0.29 with the latest Firefox version and Geckodriver 0.14 with Firefox version 50.0.2. 

Starting on Selenium 4 Rc2, it seems that when I try to start Geckodriver + Firefox it doesn't work unless it's on Geckodriver version 0.30.

I am not sure if this should be considered a regression because I am not using the latest driver, but it used to work on Selenium 4 Rc1 and now it doesn't, so maybe it's a regression?

## Last working Selenium version

Worked up to version: Selenium 4.0.0 RC 1

Stopped working in version: Selenium 4.0.0 RC 2

## To Reproduce

Detailed steps to reproduce the behavior:

E.g. on this code:

```
using OpenQA.Selenium.Firefox;
using System;

namespace Selenium_RC2_Firefox
{
    class Program
    {
        static void Main(string[] args)
        {
            FirefoxOptions _firefoxOptions = new FirefoxOptions();
            FirefoxDriverService driverService = FirefoxDriverService.CreateDefaultService(".", "geckodriver.exe"); 
            driverService.InitializationTimeout = TimeSpan.FromSeconds(30);
            _firefoxOptions.AcceptInsecureCertificates = true;

            try
            {
                var Browser = new FirefoxDriver(driverService, _firefoxOptions);
            }
            catch ( Exception e)
            {

            }
        }
    }
}
```

When running on Geckodriver 0.30 it works.
When running on Geckodriver 0.29 it doesn't work and gives this error.

![image](https://user-images.githubusercontent.com/6776230/135648865-df427364-d682-4442-b845-a794a6e091ae.png)

## Expected behavior
Driver was supposed to start and open firefox.

## Environment

OS: Windows 10
Browser: Firefox Latest Version
Browser version:  92.0.1 for Geckodriver 0.29 and 0.30 and 50.0.2 for Geckodriver 0.14.
Browser Driver version: Geckodriver 0.29, Geckodriver 0.30 and Geckodriver 0.14
Language Bindings version: .NET Core 2.1

**Comment by jimevans** [comment]
Updating geckodriver is the way to go here. Selenium now adds an additional command line argument for starting geckodriver to enable bidirectional communication with the driver so that development can proceed on implementing the [W3C WebDriver Bidi](https://w3c.github.io/webdriver-bidi/) spec. Previous versions of geckodriver don't understand the command line argument.

**Comment by diemol** [comment]
Closing as the fix to upgrade to GeckoDriver 0.30.0
